examples/sse/README.md

```diff
@@ -0,0 +1,42 @@
+## SSE example
+
+Simple `Server Send Events` example with `kstreams`
+
+### Requirements
+
+python 3.8+, poetry, docker-compose
+
+### Installation
+
+```bash
+poetry install
+```
+
+## Usage
+
+1. Start the kafka cluster: From `kstreams` project root execute `./scripts/cluster/start/`
+2. Start the ` FastAPI webserver`. Inside the `sse` folder execute `PYTHONPATH=src python -m src`
```

Comment:
If you are using poetry it automatically detects the `src` path. Normally you would put modules inside src:

```sh
src/sse/app.py
src/sse/streaming/streams.py
```

```python
from sse.streaming import foo
```